index.html

```diff
@@ -26,6 +26,165 @@ <h1 class="career">I'm a software developer</h1>
+                <div class="about-cont">
+                    <span></span>
+                    <h2 class="about-me-title">About me</h2>
+                    <img class="mask1" src="images/Mask_Group.png" alt="mask1">
+                </div>
+                <p class="about-me-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                <button class="get-my-resume">Get my resume</button>
+            </section>
+            <section class="container-skills">
+                <img class="icon-lang" src="images/icon-languages.svg" alt="icon-languages">
+                <h2 class="languages-title">Languages</h2>
+                <section class="buttons">
+                    <button class="icons">JavaScript</button>
+                    <button class="icons">Ruby</button>
+                    <button class="icons">Html</button>
+                    <button class="icons">CSS</button>
+                </section>
+            </section>
+            <section class="container-skills">
+                <img class="icon-lang" src="images/icon-frameworks.svg" alt="icon-languages">
+                <h2 class="languages-title">Frameworks</h2>
+                <section class="buttons">
+                    <button class="icons-frame">Bootstrap</button>
+                    <button class="icons-frame">Ruby on rails</button>
+                    <button class="icons-frame">RSpec</button>
+                    <button class="icons-frame">Capybara</button>
+                    <button class="icons-frame">Selenium</button>
+                </section>
+            </section>
+            <section class="container-skills">
+                <img class="icon-lang" src="images/icon-skills.svg" alt="icon-languages">
+                <h2 class="languages-title">Skills</h2>
+                <section class="buttons">
+                    <button class="icons-skills">Codekit</button>
+                    <button class="icons-skills">GitHub</button>
+                    <button class="icons-skills">Codepen</button>
+                    <button class="icons-skills">Gitlab</button>
+                    <button class="icons-skills">Terminal</button>
+                </section>
+            </section>
+            <section class="final-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</section>
+            <section class="data">
+                <input class="input-name" type="text" placeholder="Full name">
+                <input class="email-address" type="email" placeholder="Email address">
+                <textarea name="text1" class="text1" cols="30" rows="10">Enter text here</textarea>
+                <section class="side-im">
+                    <img class="side-form" src="images/form.svg" alt="side-form">
+                </section>
```

Comment:
- [ ] OPTIONAL: I strongly suggest you to render this image as a background image from css. It will make a huge improvement on your page. Currently some items are not centred because you are trying to use `flexbox` to align the image and content. This is a wrong approach and will make more issues in. the future. Please consider rendering background images from `css` since they are not part of `html`